jobs/SampleJsBuildJob.groovy

```diff
@@ -1,20 +1,26 @@
 import JsJobBuilder
+import FrontEndTestJobBuilder
 
 
 String basePath = 'JsJobSamples'
 List developers = ['irina.muchnik@cfpb.gov', 'daniel.davis@cfpb.gov']
+
 def repos = [
-        [name: 'jenkins-automation', url: "https://github.com/cfpb/jenkins-automation"],
+        [name: 'jenkins-automation', url: "https://github.com/cfpb/jenkins-automation@2.0"],
         [name: 'collab', url: "https://github.com/cfpb/jenkins-automation"]
 ]
 folder(basePath) {
```

Comment:
folder is good, but our convention has been to use views more.  Can we change this to show using a view configured with a specific regex instead?